code/__HELPERS/roundend.dm

```diff
@@ -711,3 +714,23 @@
 		Q.Execute()
 		qdel(Q)
 
+/datum/controller/subsystem/ticker/proc/antagrounds()
+	var/list/viableminds = list()
+	var/numberofrounds = 1
+	for(var/datum/antagonist/A in GLOB.antagonists)
+		if(!A.restrictmultiplerounds)
+			continue
+		if(!A.owner)
+			continue
+		viableminds += A.owner
+	
+	if(GLOB.startingminds < CONFIG_GET(number/minimal_access_threshold)) /// Low pop check
```

Comment:
I think we probably only want to do this on low pop rounds... the issue is that on low pop its always the same 1 or 2 people that have antag enabled